script.js

```diff
@@ -24,25 +30,19 @@ function updateSummaryTable() { //here we fill our summaryTableData with numbers
   fetch(summaryRequest, requestOptions)
     .then((response) => response.json())
     .then((result) => {
-        //console.log(result);
-       
-        //is there a better way to fill the summary table???
-        document.getElementById("casesWorldwide").innerText = result.Global.TotalConfirmed.toLocaleString();
-        document.getElementById("deathsWorldwide").innerText = result.Global.TotalDeaths.toLocaleString();
-        document.getElementById("casesEU").innerText = "??";
-        document.getElementById("deathsEU").innerText = "??";
-        document.getElementById("casesGermany").innerText = result.Countries[63].TotalConfirmed.toLocaleString();
-        document.getElementById("deathsGermany").innerText = result.Countries[63].TotalDeaths.toLocaleString();
-        document.getElementById("casesBavaria").innerText = Number(0).toLocaleString();
-        document.getElementById("deathsBavaria").innerText = Number(0).toLocaleString();
-        // summaryTableData.casesWorldwide = result.Global.TotalConfirmed.toLocaleString();
-        // summaryTableData.deathsWorldwide = result.Global.TotalDeaths.toLocaleString();
-        // summaryTableData.casesEU = "??";
-        // summaryTableData.deathsEU = "??";
-        // summaryTableData.casesGermany = result.Countries[63].TotalConfirmed.toLocaleString();
-        // summaryTableData.deathsGermany = result.Countries[63].TotalDeaths.toLocaleString();
-        // summaryTableData.casesBavaria = 0;
-        // summaryTableData.deathsBavaria = 0;
+      //console.log(result);
+
+      let summaryTableData = {}; //creating a dictionary
```

Comment:
all this is equal to:
```js
const summaryTableData = {
  casesWorldwide: result.Global.TotalConfirmed.toLocaleString(),
  deathsWorldwide: result.Global.TotalDeaths.toLocaleString(),
  casesEU: "??",
  deathsEU: "??",
  casesGermany: result.Countries[63].TotalConfirmed.toLocaleString(),
  eathsGermany: result.Countries[63].TotalDeaths.toLocaleString(),
  casesBavaria:  Number(0).toLocaleString(),
  deathsBavaria:  Number(0).toLocaleString(),
};
```

Reply:
and another question: why you need to explicitly cast everything into a strings?

Reply:
To make it look prettier. Hier ist how the big numbers look without usage of the `toLocalString()` method:
![image](https://user-images.githubusercontent.com/7361924/97451826-6cd7ae00-1934-11eb-8716-2a445f3b4455.png)
And hier after using this method:
![image](https://user-images.githubusercontent.com/7361924/97452047-a01a3d00-1934-11eb-9bc1-1e5ddb9e062d.png)